### PR TITLE
Implement jobs task detail

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
ℹ️   _This branch depends on features that will be introduced with ~~#494~~, thus it will fail until the respective branches get merged._

---
Add the necessary routes and links for the jobs task detail view.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/routes/jobs.js [hunk 1]
```diff
@@ -19,7 +27,42 @@ let jobsRoutes = {
           handler: JobDetailPage,
           name: 'jobs-page-detail',
           path: ':id/?',
-          hideHeaderNavigation: true
+          hideHeaderNavigation: true,
```
Comment (rcorral) [inline src/js/routes/jobs.js]: @orlandohohmeier  will you be adding the breadcrumbs in a separate PR?

Comment (orlandohohmeier) [inline src/js/routes/jobs.js]: @rcorral Yep, I already prepared one...

**Comment by kennyt** [comment]
can you post a screenshot of integration tests?

**Comment by rcorral** [comment]
img

File: src/js/pages/jobs/JobDetailPage.js [hunk 2]
```diff
@@ -232,6 +233,10 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
       return this.getLoadingScreen();
     }
 
+    if (this.props.params.taskID) {
+      return <RouteHandler />;
```
Comment (mlunoe) [inline src/js/pages/jobs/JobDetailPage.js]: I know I am late to the table, but we should really create routes for this.
